api/shorelineApi.go

```diff
@@ -546,6 +362,7 @@ func (a *Api) DeleteUser(res http.ResponseWriter, req *http.Request, vars map[st
 // status: 200 TP_SESSION_TOKEN,
 // status: 400 STATUS_MISSING_ID_PW
 // status: 401 STATUS_NO_MATCH
+// status: 403 STATUS_NOT_AUTHENTICATED
```

Comment:
This is why I asked you about this word. 

A 403 is the right return code here -- forbidden -- but the word "Authenticated" is misleading to the recipient. "Verified" would be a better word. I don't know if this is strewn throughout our code base yet or not -- but if it's possible to change it, I would prefer that we did. I think it might be just here and in the go-common change.